### ISSUE TITLE
Config files are created as directories after first docker-compose up

### ISSUE THREAD

**Issue by mariorguezari** [AriHealth/elk-secure-docker-compose#1]
## Expected Behavior

Expected containers deployed after docker-compose up:
  Starting keycloak-proxy ... done
  Starting kibana         ... done
  Starting elastalert     ... done
  Starting logstash       ... done
  Starting keycloak       ... done
  Starting mysql          ... done
  Starting elasticsearch  ... done
  Starting nginx-proxy    ... done

## Actual Behavior

  Starting elastalert     ... error

ERROR: for elastalert Cannot start service elastalert: OCI runtime create failed: container_linux.go:345: starting container process caused "process_linux.go:424: container init caused \"rootfs_linux.go:58: mounting \\\"/home/docker/elk/elastalert/config/elastalert.yaml\\\" to rootfs \\\"/var/lib/docker/overlay2/799739c996548343b49abd98b55527ba00b9ecfd20dd7f67e756bfca0b0a7d07/merged\\\" at \\\"/var/lib/docker/overlay2/799739c996548343b49abd98b55527ba00b9ecfd20dd7f67e756bfca0b0a7d07/merged/opt/elastalert/config.yaml\\\" caused \\\"not a directory\\\"\"": unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type

## Steps to Reproduce the Problem
  
  1. Vagrantfile with ubuntu xenial 
  1. git clone <<this repo>>
  1.  docker-compose up

## Specifications

  - Version: Docker version 18.09.6, build 481bc77
  - Platform: Ubuntu xenial running in Vagrant

**Comment by mariorguezari** [comment]
Manual solution: 
1. Edit .env file, that must be inside a home directory, and update CONFIG_FOLDER=/home/docker with the repo path.
1. Delete the directories that caused the error, especified in the error descrition.
1. Copy with sudo elasticsearch (and/or kibana) config folders in the path that caused the error. 
1. docker-compose down
1. docker-compose up

**Comment by carloscaverobarca** [comment]
Here we have two different issues:
- We are sharing volumes with the configuration files stored in the repo to properly launch the containers
- MySQL and Elastic need some shared volumes in the host server (Docker)

Possible solution:
- CONFIG_FOLDER should be used to configure the the output shared volumes (MySQL and Elastic)
- The input shared volumes with the configuration files must use relative paths to avoid the error

**Comment by mariorguezari** [comment]
About CONFIG_FOLDER, maybe the solution could be using the current folder where the repo was cloned:
CONFIG_FOLDER=.

and placing the config files directly in a new folder in the repo: elk/

**Comment by carloscaverobarca** [comment]
I would remove any reference of elk/ in the .env file, then everything would work. The problem is that the output shared folder would be created at the git level. We need to describe this in the README so users can create the shared folders in other places.

**Comment by mariorguezari** [comment]
Yes, the solution could be avoiding that elk folder (notice that elk/ is configured in docker-compose.yaml file), and, for the momment, accepting that the repository is cloned in the place in which you want the shared folder.
